  0%|                                                                                               | 0/1 [00:00<?, ?it/s]/home/users/delon/aemulusnu_massfunction/utils.py:36: IntegrationWarning: The occurrence of roundoff error is detected, which prevents 
  the requested tolerance from being achieved.  The error may be 
  underestimated.
  res, err = quad(dσ2dk, 0, 20/R, limit=1000)
/home/users/delon/aemulusnu_massfunction/utils.py:60: IntegrationWarning: The occurrence of roundoff error is detected, which prevents 
  the requested tolerance from being achieved.  The error may be 
  underestimated.
  res, err = quad(dσ2dRdk, 0, 20/R, limit=1000)
100%|███████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:14<00:00, 14.25s/it]100%|███████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:14<00:00, 14.25s/it]
Traceback (most recent call last):
  File "/home/users/delon/aemulusnu_massfunction/fit_individ_iter.py", line 172, in <module>
    with open("/oak/stanford/orgs/kipac/users/delon/aemulusnu_massfunction/%s_%.2f_params.pkl"%(prev_box, prev_a), "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/oak/stanford/orgs/kipac/users/delon/aemulusnu_massfunction/Box49_1400_0.30_params.pkl'
